### PR TITLE
Fix ws_transaction_command_list

### PR DESCRIPTION
**Pull request by neithernut**
Should be obvious. We want statements, not commands. Statements are command invocations.
Targets #233.

### CODE REVIEW

File: src/objects/message/transaction.h [hunk 1]
```diff
@@ -50,7 +50,7 @@ enum ws_transaction_flags {
  */
 struct ws_transaction_command_list {
     size_t n; //!< @protected length of the command array
-    struct ws_command* cmds; //!< @protected Command objects
+    struct ws_statements* statements; //!< @protected Statements of the transaction
```
Comment (matthiasbeyer) [inline src/objects/message/transaction.h]: `ws_statements` Sounds wrong as typename here.

Comment (neithernut) [inline src/objects/message/transaction.h]: Will fix _as soon as I'm done with my current task_.

Comment (neithernut) [inline src/objects/message/transaction.h]: Interupted my work to fix this.

**Comment by neithernut** [comment]
Info: I'm waiting for this, too. So Can I get an ACK already?

**Comment by matthiasbeyer** [comment]
ACK.

**Pull request merged**